main.py

```diff
@@ -25,7 +26,10 @@ def main(input_code, input_lang, output_lang):
     elif input_lang == "java":
         gast = java_main.java_to_gast(input_code)
     elif input_lang == "bash":
-        gast = bash_main.bash_to_gast(input_code)
+        process = subprocess.Popen(['node', 'bash/code_to_gast/bash_main.js', input_code], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Would like this logic to be abstracted away into a function with the same signature as the other ones.

Reply:
The problem is that this is python code calling javascript code, so I can't put the logic in bash_main because it is a js file. I could make a helper in main.py but that seems awkward

Reply:
Why would a helper be awkward? Doesn't need to be in `main.py`.